### ISSUE TITLE
Service injections not available in models

### ISSUE THREAD

**Issue by jembezmamy** [orbitjs/ember-orbit#160]
I'd like to inject `I18n` service in my model, but it seems that models are initialized via `create` method (not via `owner.lookup`) so injections doesn't work in them.

Is it made on purpose? Maybe I'm trying to do something against Ember conventions.

```javascript
// models/item.js
import { Model } from 'ember-orbit';
import { inject as service } from "@ember/service"

export default Model.extend({
  i18n: service()
});
```

```javascript
store.find("item", 1).then((item) => item.get("i18n"));
// error: Assertion Failed: Attempting to lookup an injected property on an object without a container, ensure that the object was instantiated via a container.
```

**Comment by jembezmamy** [comment]
And now I've got the same problem with [ember-cp-validations](https://github.com/offirgolan/ember-cp-validations) - I'd like to mix in validations into my ember-orbit model. Unfortunately it's not possible for the same reason as I've described before - I get this error message: `[ember-cp-validations] 'lookupValidator' requires owner/container access`.

@dgeb I could try to fix this problem but could you please review this issue first? I mean, I'm not sure if my thinking is right and I'd love to hear your opinion before I start coding anything.

**Comment by dgeb** [comment]
@jembezmamy As per #169, models are now instantiated via the store's owner's DI configuration, which allows for service injections and more. This has been published in v0.15.8. Thanks for reporting the issue and for your patience.

**Comment by jembezmamy** [comment]
Nice, thanks!
